scripts/test/Muon/utilities/muon_workspace_wrapper_test.py

```diff
@@ -214,149 +195,5 @@ def test_that_setting_a_new_workspace_removes_the_previous_one_from_the_ADS(self
         self.assertFalse(simpleapi.mtd.doesExist("name1"))
 
 
-class MuonWorkspaceAddDirectoryTest(unittest.TestCase):
```

Comment:
do we need these tests?

Reply:
These tests are to do with creating nested directories of workspace groups. This is functionality that has been removed as doesn't work with project save or workspace history.